.evergreen/secrets_handling/README.md

```diff
@@ -36,11 +36,11 @@ If using a Linux host on Evergreen, the shorthand version of the script can be u
 
 ```yaml
 - command: subprocess.exec
-    params:
+  params:
```

Comment:
This indentation fix was missed in The https://github.com/mongodb-labs/drivers-evergreen-tools/commit/e17c99a96c0dc8620607f5929052267211d35946.